GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1b_parquet.yaml

```diff
@@ -0,0 +1,9 @@
+subclass_name: dc2_dm_catalog.DC2DMCatalog
+base_dir: /global/projecta/projectdirs/lsst/production/DC2_ImSim/Run2.1i/dpdd/calexp-v1:coadd-dr1b-v1/object_table_summary
+schema_filename: dpdd_schema.yaml
+filename_pattern: 'dpdd_object_tract_\d+\.parquet$'
+description: DC2 Run 2.1i DPDD Object Table
```

Comment:
Maybe we should say something like this is identical to `dc2_object_run2.1i_dr1b` but just with a different underlying data format to reduce confusion?